pkgs/games/warsow/engine.nix

```diff
@@ -24,6 +24,10 @@ in stdenv.mkDerivation (libs // rec {
     libpng
   ];
 
+  prePatch = stdenv.lib.optional stdenv.isAarch64 ''
```

Comment:
how about `prePatch = stdenv.lib.optional !stdenv.isx86` ?

Reply:
@multun could be, not sure about other targets such as darwin. But it doesn't build on ```aarch64``` anyway. So maybe we should exclude it, I'm pretty sure the game won't run on arm anyway.

Reply:
Marked as broken on aarch64.